packages/mco/components/mco-dashboard/data-policy/cluster-app-card/cluster-app-card.tsx

```diff
@@ -161,25 +197,30 @@ export const ClusterAppCard: React.FC = () => {
                 setCluster={setCluster}
                 setAppSet={setAppSet}
               />
-              <CardTitle className="mco-cluster-app__text--margin-top">
+              <CardTitle className="mco-cluster-app__text--margin-top mco-dashboard__subtitle--size">
                 {cluster}
               </CardTitle>
             </div>
           </CardHeader>
           <CardBody>
-            {!appSet.namespace && appSet.name === ALL_APPS ? (
-              <ClusterWiseCard
-                clusterName={cluster}
-                pvcSLAData={pvcSLAData}
-                csvData={csvData}
-                clusterResources={drClusterAppsMap}
-              />
+            {!!drClusterAppsMap?.[cluster]?.protectedAppSets?.length ? (
```

Comment:
why to block cluster view when no protected apps are present ?
we can still show cluster wise card... user can see the cluster health and that there are "0" apps etc etc...
block app wise card in case no protected apps...
```suggestion
            {!!Object.keys(drClusterAppsMap)?.length ? (
```

Reply:
i am little concern about QE, they will start raise a bug like this card empty and that card is empty :), but ya i will think about this.

Reply:
if no clusters in cluster dropdown >>  we show "No data available" at cluster card level...
if there are clusters, but no applications >> we still show cluster card, we only show "No data available" at app card level...

cluster card will not have missing info... just there are no protected apps, that is why they see count as "0" everywhere... we need to test this once...